zebra-rpc/src/methods.rs

```diff
@@ -1510,11 +1510,23 @@ pub struct AddressBalance {
 
 /// A hex-encoded [`ConsensusBranchId`] string.
 #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash, serde::Serialize, serde::Deserialize)]
-struct ConsensusBranchIdHex(#[serde(with = "hex")] ConsensusBranchId);
+pub struct ConsensusBranchIdHex(#[serde(with = "hex")] ConsensusBranchId);
+
+impl ConsensusBranchIdHex {
+    /// Returns a new instance of ['ConsensusBranchIdHex'].
+    pub fn new(consensus_branch_id: u32) -> Self {
+        ConsensusBranchIdHex(consensus_branch_id.into())
+    }
+
+    /// Returns a reference to the ['ConsensusBranchId'].
```

Comment:
```suggestion
    /// Returns the value of the ['ConsensusBranchId'].
```

Reply:
Brilliant! I have added the suggested in [308c9c6].